src/main/routes/postcode-lookup.ts

```diff
@@ -0,0 +1,26 @@
+import * as config from 'config'
+import * as express from 'express'
+import { PostcodeInfoClient, PostcodeInfoResponse } from '@hmcts/postcodeinfo-client'
+
+import { Paths as AppPaths } from 'app/paths'
+import { request } from 'client/request'
+import * as Logging from '@hmcts/nodejs-logging'
+
+const postcodeClient = new PostcodeInfoClient(config.get<string>('postcodeLookup.apiKey'), request)
+const logger = Logging.getLogger('postcode-lookup')
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(AppPaths.postcodeLookupProxy.uri, (req, res) => {
+    postcodeClient.lookupPostcode(req.query.postcode)
+      .then((postcodeInfoResponse: PostcodeInfoResponse) => res.json(postcodeInfoResponse))
```

Comment:
What do you think about using `async`/`await` for consistency?

Reply:
I think each have their part and should be used as needed, I don't think changing it gives any value.

Reply:
I actually prefer the .then syntax as was thinking of suggesting we move to it although that would be quite a big change

Reply:
@robertparkinson we were using the promise API syntax initially and then decided to move to async/await, that's the reason for my comment initial comment on this line :)